doc/data/messages/i/invalid-enum-extension/good.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class A(Enum):
+    x = 1
+    y = 2
+    z = 3
```

Comment:
```suggestion


class A(Enum):
    x = 1
    y = 2


class B(Enum):
    x = 1
    y = 2
    z = 3
```
I suppose if you need two classes the solution is to duplicate ?